common/Windows/SecondaryWindow.cs

```diff
@@ -175,19 +175,25 @@ public bool IsTopLevel
     }
 
     public SecondaryWindow()
-    {
+    {
+        // Set the theme of the secondary window before creating the window template.
+        // This must be done before the creation of the template, or else the secondary window
+        // will continue to follow the Windows system theme, and not respect the theme
+        // set by the app itself.
+        UseAppTheme = true;
+
         // Initialize window content template
         _windowTemplate = new (this);
         WindowContent = _windowTemplate;
```

Comment:
Because we're not using the default window layout from the derived class (we have our own layout and title bar), we can simply replace `WindowContent = ...` to `Content = ...`. I expect `UseAppTheme = true` to update the requested theme of the secondary window root element (which is the secondary window template).

Related code:
- [Source code](https://github.com/dotMorten/WinUIEx/blob/main/src/WinUIEx/WindowEx.cs#L179) (note we're using an older code version, but the idea is the same)

Reply:
spoke to Amir offline, we're keeping as is in the PR.